xbmc/addons/AddonCallbacksAddon.cpp

```diff
@@ -543,6 +543,7 @@ static void CFileItemListToVFSDirEntries(VFSDirEntry* entries,
     entries[i].label = strdup(items[i]->GetLabel().c_str());
     entries[i].path = strdup(items[i]->GetPath().c_str());
     entries[i].size = items[i]->m_dwSize;
+    //entries[i].mtime = items[i]->m_dateTime;
```

Comment:
uhm rewriting history? stuff came from me remember. why do you think it was there in the first place?

upstream in all its viceness removed my datetime stuff from kodiplatform. that is the only rebase error involved. it was removed from kodi_vfs_types because it was unused.